tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## April 25, 2023
+
+**General changes**
+
+- For the first five Serverless Tier clusters in your organization, TiDB Cloud provides a free usage quota for each of them as follows:
+
+    - Row storage: 5 GiB
+    - [Request Units (RUs)](/tidb-cloud/tidb-cloud-glossary.md#request-unit): 50 million RUs per month
+
+    Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount off. After that, usage beyond the free quota will be charged.
```

Comment:
```suggestion
  Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount off. After that, usage beyond the free quota will be charged.
```

Reply:
It seems that the original indentation works well?

Reply:
The original indentation won't work well in the doc website because there is an indented list before it😂